classic/src/view/form/Print.js

```diff
@@ -33,6 +33,24 @@ Ext.define("MoMo.client.view.form.Print", {
 
     controller: 'form.print',
 
+    bodyPadding: 0,
+
+
+    bbar: [{
+        xtype: 'button',
+        name: 'createPrint',
+        bind: {
+            text: '{printFormat:uppercase} {printButtonSuffix}'
+        },
+        formBind: true,
+        handler: function(btn){
```

Comment:
For the same reasons as @annarieger stated below. We need to have access to certain variables not usually available in the controller. We will probably refactor the controllers soon so we dont need them any longer :wink: